### PR TITLE
add focus to apps

### PR DESCRIPTION
**Pull request by tangxiangpi**
a new android mastodon app:
https://play.google.com/store/apps/details?id=allen.town.focus.mastodon&gl=jp

### CODE REVIEW

**Comment by Gargron** [comment]
From screenshots it looks like a Twitter app?

**Comment by tangxiangpi** [comment]
> From screenshots it looks like a Twitter app?

Based on my own app (focusTwitter), but don't worry it's already a highly customizable mastodon app and adds some features that are not officially available

**Comment by andypiper** [comment]
Relates to #354 - merging will close both that issue and this PR. In review.

File: data/apps.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ import tuba from "../public/apps/tuba.png"
 import mona from "../public/apps/mona.png"
 import fedica from "../public/apps/fedica.png"
 import type { StaticImageData } from "next/image"
+import focus from "../public/apps/focus.png"
```
Comment (andypiper) [inline data/apps.ts:28]: please move this to the end of the list of app icons, above the `import type { StaticImageData ...` line

File: data/apps.ts [hunk 2]
```diff
@@ -45,6 +46,12 @@ export type appsList = {
 }
 export const apps: appsList = {
   android: [
+      {
+      released_on: "May 21, 2023",
+      name: "Focus",
+      icon: focus,
+      url: "https://play.google.com/store/apps/details?id=allen.town.focus.mastodon",
+    },
```
Comment (andypiper) [inline data/apps.ts:54]: Please specify whether this is paid or free (`paid: true|false`)

**Review by andypiper — CHANGES_REQUESTED** [review]
Please look at the review comments and I think this can be merged.

File: data/apps.ts (hunk 2 above)
Comment (allentown521) [reply data/apps.ts:56]: changed , please merge it

File: data/apps.ts (hunk 1 above)
Comment (allentown521) [reply data/apps.ts:27]: changed , please merge it

**Review by allentown521 — APPROVED** [review]
changed

**Comment by andypiper** [comment]
Closes #354

**Review by andypiper — APPROVED** [review]

**Comment by andypiper** [comment]
Thank you!

**Pull request merged**